recipes/natural_language_processing/codegen/README.md

```diff
@@ -22,111 +31,131 @@ In order to build this application we will need a model, a Model Service and an
+Please refer to the section below for more details about [interacting with the codegen application](#interact-with-the-ai-application).
+
+To stop and remove the Pod, run:
+
+```
+podman pod stop codegen
+podman pod rm codgen
+```
+
+## Download a model
 
 If you are just getting started, we recommend using [Mistral-7B-Instruct](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1). This is a well
 performant mid-sized model with an apache-2.0 license. In order to use it with our Model Service we need it converted
 and quantized into the [GGUF format](https://github.com/ggerganov/ggml/blob/master/docs/gguf.md). There are a number of
 ways to get a GGUF version of Mistral-7B, but the simplest is to download a pre-converted one from
 [huggingface.co](https://huggingface.co) here: https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF.
 
-There are a number of options for quantization level, but we recommend `Q4_K_M`.
+There are a number of options for quantization level, but we recommend `Q4_K_M`. 
 
 The recommended model can be downloaded using the code snippet below:
 
 ```bash
-cd models
-wget https://huggingface.co/TheBloke/Mistral-7B-Code-16K-qlora-GGUF/resolve/main/mistral-7b-code-16k-qlora.Q4_K_M.gguf
-cd ../
+cd ../../../models
+curl -sLO https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf
+cd ../recipes/natural_language_processing/codgen
 ```
 
-_A full list of supported open models is forthcoming._
+_A full list of supported open models is forthcoming._  
 
 
-### Build the Model Service
+## Build the Model Service
 
-The complete instructions for building and deploying the Model Service can be found in the [the llamacpp_python model-service document](../model_servers/llamacpp_python/README.md).
+The complete instructions for building and deploying the Model Service can be found in the
+[llamacpp_python model-service document](../../../model_servers/llamacpp_python/README.md).
 
-The Model Service can be built from the root directory with the below code snippet:
+The Model Service can be built from make commands from the [llamacpp_python directory](../../../model_servers/llamacpp_python/).
 
 ```bash
-cd model_servers/llamacpp_python
-podman build -t llamacppserver -f base/Containerfile .
+# from path model_servers/llamacpp_python from repo containers/ai-lab-recipes
+make build
 ```
+Checkout the [Makefile](../../../model_servers/llamacpp_python/Makefile) to get more details on different options for how to build.
 
+## Deploy the Model Service
 
-### Deploy the Model Service
+The local Model Service relies on a volume mount to the localhost to access the model files. It also employs environment variables to dictate the model used and where its served. You can start your local Model Service using the following `make` command from `model_servers/llamacpp_python` set with reasonable defaults:
```

Comment:
Not sure about the wording of this first sentence. Localhost typically comes with the connotation of the browser / internet, where as the volume mount you are discussing is at a container level.